kornia/color/gray.py

```diff
@@ -1,44 +1,43 @@
-import torch
-import torch.nn as nn
-
 from kornia.color.rgb import bgr_to_rgb
+from kornia.core import Image, ImageColor, Tensor, concatenate, nn
 
 
-def grayscale_to_rgb(image: torch.Tensor) -> torch.Tensor:
+def grayscale_to_rgb(image: Image) -> Image:
```

Comment:
shouldn't input argument type be `Union[Image, Tensor]` ?

Reply:
in practice `Image` is a subclass of tensor. Having union it turns out that torchscript cannot resolve well